superset/security.py

```diff
@@ -130,8 +131,13 @@ def datasource_access_by_name(
 
         datasources = ConnectorRegistry.query_datasources_by_name(
             db.session, database, datasource_name, schema=schema)
+        print(datasources)
```

Comment:
Nit. Remove print statements.

Reply:
Ooops, I need to add a commit hook that checks for these.